src/ui/components/Header/UserOptions.tsx

```diff
@@ -41,6 +41,19 @@ function UserOptions({ setModal, noBrowserItem }: UserOptionsProps) {
     setModal("settings");
   };
 
+  const preloadIcons = () => (
+    <div style={{ display: "none" }}>
+      <Icon filename="docs" className="bg-iconColor" />
+      <Icon filename="help" className="bg-iconColor" />
+      <Icon filename="settings" className="bg-iconColor" />
+      <Icon filename="replay-logo" className="bg-iconColor" />
+    </div>
+  );
+
+  {
+    preloadIcons();
```

Comment:
:hand: For the record I don't think this line does anything :)

You're calling the function, which returns JSX...

but that JSX isn't being returned from the _component_.

So, React will never look at it or render those `<Icon>` components.